ServiceTier/Services/PresetService.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Common.ModelsDTO;
+using Timer.DAL.Extensions;
+using Timer.DAL.Timer.DAL.Entities;
+using Timer.DAL.Timer.DAL.UnitOfWork;
+
+namespace ServiceTier.Services
+{
+    public class PresetService : BaseService, IPresetService
+    {
+        public PresetService(IUnitOfWork unitOfWork) : base(unitOfWork)
+        {
+
+        }
+
+        public void CreatePreset(PresetDTO presetDTO)
+        {
+            Preset preset = presetDTO.ToPreset();
+            unitOfWork.Presets.Create(preset);
+            unitOfWork.Save();
+            var presets = unitOfWork.Presets.GetAll();
+            foreach (var timer in presetDTO.Timers)
```

Comment:
how about using batch insert instead?

Reply:
i dont realy understand what batch insert means in c#.I know than in sql you can just insert your values splited with coma but here i dont realy understand how it must look like